vision/src/autogluon/vision/image_prediction/image_prediction.py

```diff
@@ -51,17 +65,19 @@ def fit(self,
             For more details of how to construct a dataset for image predictor, check out:
             `http://preview.d2l.ai/d8/main/image_classification/getting_started.html`.
             If a string is provided, will search for k8 built-in datasets.
-        val_data : pd.DataFrame or str, default = None
-            Training data, can be a dataframe like image dataset.
+        tuning_data : pd.DataFrame or str, default = None
+            Another dataset containing validation data reserved for tuning processes,
```

Comment:
"reserved for tuning processes" -> "reserved for model selection and hyperparameter-tuning"